system/lib/compiler-rt/lib/ubsan_minimal/sanitizer_atomic.h

```diff
@@ -0,0 +1,226 @@
+//===-- sanitizer_atomic.h --------------------------------------*- C++ -*-===//
```

Comment:
Is this file taken verbatim from compiler-rt?

Reply:
If so which version of compiler-rt did you import it from?   Might be worth mentioning in the description.

Reply:
Can mention those details in a readme.txt file in the directory, I think that's what we do in other bundled code.